api/agent/lb_agent.go

```diff
@@ -173,7 +173,7 @@ func (a *lbAgent) GetCall(opts ...CallOpt) (Call, error) {
 
 	c.ct = a
 	c.stderr = common.NoopReadWriteCloser{}
-	c.slotHashId = getSlotQueueKey(&c)
+	c.slotHashId = getSlotQueueKey(&c, nil)
```

Comment:
This line above shows that LB is actually setting the slot key. In the past, we wanted to move slot key generation to LBs which creates some opportunities to pick/optimize runners. But at the end, we didn't utilize this much at all. So, we have to options:
1) remove this line above completely and let runners set the slot hash id
2) or make sure LB processes slot hash id correctly, (aka do not pass "nil" as driver)

Reply:
Will remove it, if its not used